lasagne/layers/embedding.py

```diff
@@ -18,7 +18,7 @@ class EmbeddingLayer(Layer):
     Parameters
     ----------
     incoming : a :class:`Layer` instance or a tuple
-        The layer feeding into this layer, or the expected input shape
+        The layer feeding into this layer, or the expected input shape.
```

Comment:
We don't have that dot in any of our docstrings, though. I didn't place a dot because it's not a full sentence, but I capitalized the first letter because it didn't look well in Sphinx otherwise. We should probably agree on what we want, but not change it in a single place!